__tests__/app.test.js

```diff
@@ -66,50 +66,133 @@ describe("/api/reviews/:review_id", () => {
+            expect(comment).toHaveProperty("author", expect.any(String));
+            expect(comment).toHaveProperty("created_at", expect.any(String));
+            expect(comment).toHaveProperty("review_id", expect.any(Number));
+            expect(comment.review_id).toBe(2);
+          });
+          expect(comments).toBeSortedBy("created_at", { descending: true });
         });
-        expect(comments).toBeSortedBy("created_at", { descending: true });
-      });
-  });
-  it("GET 200 - responds with an empty array if the given ID exists but has no comments", () => {
-    return request(app)
-      .get("/api/reviews/1/comments")
-      .expect(200)
-      .then(({ body }) => {
-        const { comments } = body;
-        expect(comments).toEqual([]);
-      });
-  });
-  it("ERROR 404 - responds with an error if given an ID that does not exist", () => {
-    return request(app)
-      .get("/api/reviews/9999/comments")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body).toEqual({ msg: "Review Not Found" });
-      });
+    });
+    it("GET 200 - responds with an empty array if the given ID exists but has no comments", () => {
+      return request(app)
+        .get("/api/reviews/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toEqual([]);
+        });
+    });
+    it("ERROR 404 - responds with an error if given an ID that does not exist", () => {
+      return request(app)
+        .get("/api/reviews/9999/comments")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body).toEqual({ msg: "Review Not Found" });
+        });
+    });
+    it("ERROR 400 - responds with an error if given an ID that is not a number", () => {
+      return request(app)
+        .get("/api/reviews/notAnId/comments")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({ msg: "Invalid ID" });
+        });
+    });
   });
-  it("ERROR 400 - responds with an error if given an ID that is not a number", () => {
-    return request(app)
-      .get("/api/reviews/notAnId/comments")
-      .expect(400)
-      .then(({ body }) => {
-        expect(body).toEqual({ msg: "Invalid ID" });
-      });
+  describe("POST", () => {
+    it("POST 201 - posts a new comment when given a valid object with username and body properties", () => {
+      const comment = {
+        username: "mallionaire",
+        body: "This is my new comment!",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(comment)
+        .expect(201)
+        .then(({ body }) => {
+          const { postedComment } = body;
+          expect(postedComment).toHaveProperty(
+            "comment_id",
+            expect.any(Number)
+          );
+          expect(postedComment).toHaveProperty("body", expect.any(String));
+          expect(postedComment).toHaveProperty("review_id", expect.any(Number));
```

Comment:
I would defo make sure that you have the right review - you've input review_id 1 so it should definitely be 1 that is returned.